### DISCUSSION TITLE
Is there any way to fetch translations from a url?

### DISCUSSION THREAD

**Discussion by juniorforlife** [sergiodxa/remix-i18next#146] [Q&A] [answered]
Instead of storing translation files in public/locales, I’d like to fetch them from a url hosted by a dedicated BE. Is there any way to achieve this with this library?

**Comment by sergiodxa** [comment, marked as answer]
You can use the fetch backend for i18next for that, this is configurable using the i18next backends and outside the scope of remix-i18next.
